tracim/tracim/lib/email.py

```diff
@@ -67,17 +67,26 @@ def connect(self):
             logger.info(self, 'Connecting from SMTP server {}'.format(self._smtp_config.server))
             self._smtp_connection = smtplib.SMTP(self._smtp_config.server, self._smtp_config.port)
             self._smtp_connection.ehlo()
+
             if self._smtp_config.login:
                 try:
                     starttls_result = self._smtp_connection.starttls()
                     logger.debug(self, 'SMTP start TLS result: {}'.format(starttls_result))
                 except Exception as e:
                     logger.debug(self, 'SMTP start TLS error: {}'.format(e.__str__()))
+                    
+            if self._smtp_config.login:
+                try:
+                    login_res = self._smtp_connection.login(self._smtp_config.login, self._smtp_config.password)
+                    logger.debug(self, 'SMTP login result: {}'.format(login_res))
+                except Exception as e:
+                    logger.debug(self, 'SMTP login error: {}'.format(e.__str__()))
+                    raise e
```

Comment:
Do not raise again the exception: the email send process is in a background task so raised exceptions would kill the thread.